docs/source/configuration/slots.md

```diff
@@ -0,0 +1,260 @@
+---
+myst:
+  html_meta:
+    "description": "Slots are insertion points in the Volto rendering tree structure."
+    "property=og:description": "Slots are insertion points in the Volto rendering tree structure."
+    "property=og:title": "Slots"
+    "keywords": "Volto, Plone, frontend, React, configuration, slots, viewlets"
+---
+
+# Slots
+
+Slots are insertion points in the Volto rendering tree structure.
```

Comment:
How about:

```suggestion
Slots provide a way for Volto addons to insert their own components at predefined locations in the rendered page.```

(i.e. mention what problem they solve, not just how they operate)

Reply:
@davisagli your suggestion got tweaked out. I think this version is correct:

```suggestion
Slots provide a way for Volto add-ons to insert their own components at predefined locations in the rendered page.
```

Reply:
@stevepiercy oops, thanks, updated

Reply:
"add-ons" is hyphenated.